platform/jobs/edxEndToEndTestsJob.groovy

```diff
@@ -0,0 +1,70 @@
+package platform
+
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_LOG_ROTATOR
+import static org.edx.jenkins.dsl.JenkinsPublicConstants.JENKINS_PUBLIC_WORKER
+
+// The edx-e2e-tests job is run automatically on every deployment of the edx-platform
+// from the gocd pipeline.
+
+job('NEW-edx-e2e-test') {
```

Comment:
are you temporarily making this with a new job name so that we can reseed without destroying what's there now?